{"name": "test_zero_volume_cases[-CES-DAI]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 429 == 200\n +  where 429 = <Response [429]>.status_code", "trace": "tests/special/test_edge_cases.py:16: in test_zero_volume_cases\n    assert response.status_code == 200\nE   assert 429 == 200\nE    +  where 429 = <Response [429]>.status_code"}, "parameters": [{"name": "timeframe", "value": "''"}, {"name": "pair", "value": "'CES-DAI'"}], "start": 1758131936983, "stop": 1758131937233, "uuid": "c8c5f6f9-2b83-4b2d-b6dc-270d1280dfb8", "historyId": "e7a353b1b2a73f2cb8ab09e9ed8cfb18", "testCaseId": "f1061a3867aae255684da5f0a2754e42", "fullName": "tests.special.test_edge_cases#test_zero_volume_cases", "labels": [{"name": "tag", "value": "special"}, {"name": "parentSuite", "value": "tests.special"}, {"name": "suite", "value": "test_edge_cases"}, {"name": "host", "value": "826d662bff36"}, {"name": "thread", "value": "2128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.special.test_edge_cases"}], "titlePath": ["tests", "special", "test_edge_cases.py"]}